internal/config/config.go

```diff
@@ -138,6 +143,13 @@ func (c *Config) Load() error {
 	c.Feeds = fileConfig.Feeds
 	c.Openers = fileConfig.Openers
 
+	if fileConfig.HTTPOptions != nil {
+		if _, err := TLSVersion(fileConfig.HTTPOptions.MinTLSVersion); err != nil {
+			return err
```

Comment:
This doesn't seem to work as expected. I get unhelpful output like this if I set it to some unsupported string:

```
flax% go run cmd/nom/main.go
Usage:
  main [OPTIONS]

Application Options:
  -v, --verbose      Show verbose logging
  -n, --number=      Number of results to show
  -p, --pager=       Pager to use for longer output. Set to false for no pager
      --config-path= Location of config.yml
  -f, --feed=        Feed(s) URL(s) for preview
      --version      Display version information

Help Options:
  -h, --help         Show this help message
exit status 1
```

Reply:
What I would expect is to see a "config error" message and exit status 1.

Reply:
if you run `go run cmd/nom/main.go -v` do you see the error? This is a gotcha with the `--verbose` flag showing errors. Needs to be refactored out at some point